implementations/scripts/issue_github.py

```diff
@@ -0,0 +1,129 @@
+from copy import deepcopy
+
+from implementations.scripts.metric import Metric
+from implementations.scripts.utils import str_to_date
+
+
+class IssueGithub(Metric):
+    """
+    Generic class for issue metrics without pandas structure
+    """
+
+    def __init__(self, items, date_range=(None, None), reopen_as_new=False):
+        """
+        Constructor of this class
+
+        :param items: list of issues fetched by Perceval into a flat dictionary
+        :param date_range: Tuple, containing a range of date
+        :param reopen_as_new: allow to consider that reopen an issue is a new issue.
+            A limitation to keep in mind is that Perceval doesn't fetch event data,
+            and this parameter needs the event data to know if the issue has been
+            reopened. Then, the reopen_as_new parameter couldn't be worked if we
+            doesn't have the event data of issues.
```

Comment:
Thanks for the patch, @xavierbol! I'm really liking this version. Though the doc-string clearly explains what the `reopen_as_new` represents, I think the wording can be slightly improved. Maybe something like this:
```
"""
:param reopen_as_new: A criteria for deciding whether reopened issues
        are considered as new issues. If True, every time an issue is reopened,
        it is treated as a new issue.

		A limitation to keep in mind is that Perceval
		currently does not retrieve the event data necessary 
		for deciding whether or not an issue has been reopened. Thus, this 
		parameter does not work as expected and shouldn't be set to True.
"""
```